refactor(linter): use `BigIntLiteral::raw` field

overlookmotel opened the pull request:
No need to slice source text to get raw value of `BigIntLiteral`. It already has a `raw` field.

overlookmotel commented:
* **#7660** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/7660?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈 <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/7660?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.
<!-- Current dependencies on/for this PR: -->

Boshen commented:
### Merge activity

* **Dec 4, 11:09 PM EST**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc) once it meets the requirements.

overlookmotel committed:
refactor(linter): use `BigIntLiteral::raw` field (#7660)

No need to slice source text to get raw value of `BigIntLiteral`. It already has a `raw` field.
